litellm/llms/bedrock.py

```diff
@@ -702,6 +702,11 @@ def _embedding_func_single(
     encoding=None,
     logging_obj=None,
 ):
+    if type(input) != str:
```

Comment:
@rmann-nflx doing the check here as opposed to 810, because on 810 we'd have to loop through all elements in the list and assert none of them are str

Reply:
Makes sense -- much cleaner approach imho